content_zh/docs/reference/config/policy-and-telemetry/expression-language/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 表达式语言
+description: Mixer 的配置表达式语言参考。
+weight: 20
+aliases:
```

Comment:
Please remove aliases, they're only used for old files moving to a new location. This is a new file.

Reply:
done, but the english version still has 'aliases'